### ISSUE TITLE
Jest Test Failures on MacOS

### ISSUE THREAD

**Issue by nicholaswold** [nteract/archived-desktop-app#5330]
**Application or Package Used**
This is on a clean clone of the Nteract monorepo.

**Describe the bug**
On MacOS, two Jest tests are failing on a clean build of the repo:

1. `KeyboardShortcuts: Executes focused cell if the correct key combination is pressed`
1. `KeyboardShortcuts: Should not focus next cell's editor if it is markdown`

**To Reproduce**

On a MacOS machine (mine is Mojave, @willingc what about yours?)
```
yarn clean
rm -rf node_modules
yarn install
yarn build:all
yarn test
```

**Expected behavior**
That the tests succeed on MacOS machines, or are skipped if there is a reason they cannot pass.

**Screenshots**
![image](https://user-images.githubusercontent.com/1581984/96645822-03c2bb80-12e0-11eb-83d7-f0e37fc64f8b.png)

**Desktop (please complete the following information):**

- OS: MacOS
- Commit: https://github.com/nteract/nteract/commit/8d3764308730dda6a3a5f1df8e48e0ccc8efa11d

**Additional context**
@willingc you also ran into this issue, and confirmed personally and with someone else that this doesn't occur on Linux machines?

**Comment by captainsafia** [comment]
These failures do occur on macOS machines sporadically but we haven't been able to track down the cause and strengthen them here. They also don't occur on our Linux-based CI environments.

**Comment by willingc** [comment]
I'm running into these test failures consistently with Catalina 10.15.7. It did not occur on my Linux machine.

**Comment by captainsafia** [comment]
Closed via a895ddb.
